Added Core Contract Events Docs (#613)
onflow/flow: docs/content/core-contracts/flow-token.mdx

| **`FT.01`** | Transfer FLOW            | [flowToken/transfer_tokens.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/flowToken/transfer_tokens.cdc)         |
| **`FT.02`** | Get an account's balance | [flowToken/scripts/get_balance.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/flowToken/scripts/get_balance.cdc) |
| **`FT.03`** | Get total supply         | [flowToken/scripts/get_supply.cdc](https://github.com/onflow/flow-core-contracts/blob/master/transactions/flowToken/scripts/get_supply.cdc)   |

# Events

Flow relies on a set of core contracts that define key portions of the Flow protocol. Those contracts are core contracts 

> and are made to emit the events documented bellow. You can read about the core [contracts here](https://docs.onflow.org/core-contracts) 

Reviewer: ```suggestion
and are made to emit the events documented below. You can read about the core [contracts here](https://docs.onflow.org/core-contracts) 
```

> and view their source code with events definitions.

Reviewer: ```suggestion
and view their source code and event definitions.
```

> Events emitted from core contracts follow the standard format:

Reviewer: ```suggestion
Events emitted from core contracts follow a standard format:
```

```
A.{contract address}.{contract name}.{event name}
```

> The components of the event name:

Reviewer: ```suggestion
The components of the event-name format are:
```

- `contract address` - the address of the account the contract has been deployed to
- `contract name` - the name of the contract in the source code

> - `event name` - the name of the event found in the source code event definition

Reviewer: ```suggestion
- `event name` - the name of the event as declared in the source code
```

### Flow Token Contract

> Description of events emitted from the [Flow Token contract](https://docs.onflow.org/core-contracts/flow-token/). 

Reviewer: ```suggestion
Description of events emitted from the [FLOW Token contract](https://docs.onflow.org/core-contracts/flow-token/). 
```

> The contract defines the fungible Flow token. Please note that events for the fungible token contracts are the same 

Reviewer: ```suggestion
The contract defines the fungible FLOW token. Please note that events for the fungible token contracts are the same 
```

### Tokens Burned

Event that is emitted when tokens get destroyed.

- Event name: `TokensBurned`
- Mainnet event: `A.1654653399040a61.FlowToken.TokensBurned`
- Testnet event: `A.7e60df042a9c0868.FlowToken.TokensBurned`

```cadence
pub event TokensBurned(amount: UFix64)
```

| Field             | Type   | Description                                                            |
| ----------------- | ------ | ---------------------------------------------------------------------- |
| amount       | UFix64 | The amount of tokens to burn |

### Minter Created

Event that is emitted when a new minter resource gets created.

- Event name: `MinterCreated`
- Mainnet event: `A.1654653399040a61.FlowToken.MinterCreated`
- Testnet event: `A.7e60df042a9c0868.FlowToken.MinterCreated`

```cadence
pub event MinterCreated(allowedAmount: UFix64)
```

| Field             | Type   | Description                                                            |
| ----------------- | ------ | ---------------------------------------------------------------------- |
| allowedAmount       | UFix64 | The amount of tokens that the minter is allowed to mint |

### Burner Created

> Event that is emitted when a new burner resource gets created.

Reviewer: ```suggestion
Event that is emitted when a new burner Resource gets created.
```

- Event name: `BurnerCreated`
- Mainnet event: `A.1654653399040a61.FlowToken.BurnerCreated`
- Testnet event: `A.7e60df042a9c0868.FlowToken.BurnerCreated`

```cadence
pub event BurnerCreated()
```

### Staking Events

> Read about the staking events here: [https://docs.onflow.org/staking/events/](https://docs.onflow.org/staking/events/)

Reviewer: ```suggestion
TO learn more about Staking events, read [https://docs.onflow.org/staking/events/](https://docs.onflow.org/staking/events/)
```
